### ISSUE TITLE
lc0-compatibility

### ISSUE THREAD

**Issue by bjoernf73** [MGleason1/PGN-Spy#10]
According to lc0, it should be UCI-compatible. However, PGN-spy fails when used with lc0: 
![image](https://user-images.githubusercontent.com/46068634/190864974-7105a889-e4ec-4e5f-a6af-be4f2f407e11.png)
Fails immediately: 
![image](https://user-images.githubusercontent.com/46068634/190865156-3c23cb51-ad44-46f4-8f23-f61fe0995343.png)

I realize PGN-spy only works with select UCI-compatible engines - however, this would be great.

**Comment by MGleason1** [comment]
Yeah, I'm interacting with the engine in a somewhat non-standard way, and it's a little brittle.  I want to rewrite the engine interaction code from scratch, but haven't gotten around to it.

**Comment by 0x64RT** [comment]
> Yeah, I'm interacting with the engine in a somewhat non-standard way, and it's a little brittle. I want to rewrite the engine interaction code from scratch, but haven't gotten around to it.

Are you still thinking on rewritting it ?
